frontend/packages/integration-tests-cypress/tests/i18n/pseudolocalization.spec.ts

```diff
@@ -22,7 +22,6 @@ describe('Localization', () => {
     masthead.clickMastheadLink('help-dropdown-toggle');
     // wait for both console help menu items and additionalHelpActions items to load
     // additionalHelpActions come from ConsoleLinks 'HelpMenu' yaml and are not translated
```

Comment:
It looks like this comment is no longer valid, but I'm OK cleaning that up on the last PR since the logic below will need to change anyway.

Reply:
I'll update the comments in the next PR